### ISSUE TITLE
Exit button 

### ISSUE THREAD

**Issue by malliaanthony** [stevencamilleri/IMTAssignment2016#5]
I noticed that i cannot exit the scene of the game once playing such game. Is there any case where an exit button could be implemented so that the user could exit the game. For instance he/she could check what one has to do so that the level could be passed.

**Comment by stevencamilleri** [comment]
I agree with you on this, an exit button makes sense as it makes it easier for the user to check out controls/instructions or restart the game. I'll make sure to implement this.

**Comment by stevencamilleri** [comment]
https://github.com/stevencamilleri/IMTAssignment2016/commit/49839d9f7c4897ce5df1c35819651edfa8c32954 - A link to the commit where the changes have been applied.
